searchcorespi/src/vespa/searchcorespi/index/iindexmanager.h

```diff
@@ -94,7 +94,7 @@ class IIndexManager {
      * @param serialNum       The unique monotoninc increasing serial number
      *                        for this operation.
      **/
-    virtual void putDocument(uint32_t lid, const Document &doc, SerialNum serialNum) = 0;
+    virtual void putDocument(uint32_t lid, const Document &doc, SerialNum serialNum, OnWriteDoneType on_write_done) = 0;
```

Comment:
Please extend the function doc to include the new param.

Reply:
Done.